### PR TITLE
Scrapper, Ekaterina Nushkina - 21FPL1

### PR DESCRIPTION
**Pull request by eanushkina**

### CODE REVIEW

**Review by dmitry-uraev — CHANGES_REQUESTED** [review]
I see some changes, but no changes in lab_5 scrapper file. It is right time to start implementing lab_5.

**Review by dmitry-uraev — CHANGES_REQUESTED** [review]
No activity in PR.
It is right time to implement scrapper!

**Comment by dmitry-uraev** [comment]
No activity in PR.

**Comment by dmitry-uraev** [comment]
No progress since last review

**Comment by dmitry-uraev** [comment]
No progress since last review

**Comment by dmitry-uraev** [comment]
No progress

File: main.py [hunk 1]
```diff
@@ -0,0 +1,24 @@
+from pymystem3 import Mystem
```
Comment (dmitry-uraev) [inline main.py]: please remove files not related to labs assignment

**Review by dmitry-uraev — CHANGES_REQUESTED** [review]
Not seeing any activity in scrapper or pipeline. 

@eanushkina , are you sure you need review?

File: lab_5_scrapper/scrapper.py [hunk 2]
```diff
@@ -144,22 +252,36 @@ def parse(self) -> Union[Article, bool, list]:
...
     Creates ASSETS_PATH folder if no created and removes existing folder
     """
-    pass
+    if base_path.exists():
+        shutil.rmtree(base_path)
+    base_path.mkdir(parents=True)
 
 
-def main() -> None:
+    def main() -> None:
```
Comment (dmitry-uraev) [inline lab_5_scrapper/scrapper.py]: not sure you need tab/whitespaces here

Comment (eanushkina) [reply lab_5_scrapper/scrapper.py]: fixed

File: main.py (hunk 1 above)
Comment (eanushkina) [reply main.py]: fixed

**Pull request closed without merging**